js/admin.js

```diff
@@ -152,8 +152,12 @@ jQuery( document ).ready(
 		// Create the selectmenu widget only if the field is present.
 		if ( flagListExist ) {
 			// Create the jQuery UI selectmenu widget for flags list dropdown and return its instance.
-			// Need to keep the instance to be able to use it in the managment of the languages list.
-			 var selectmenuFlagList = initializeSelectmenuWidget( $( '#flag_list' ), Object.assign( {}, selectmenuOptions, selectmenuFlagListCallbacks ) );
+			var selectmenuFlagList = initializeSelectmenuWidget( $( '#flag_list' ), Object.assign( {}, selectmenuOptions, selectmenuFlagListCallbacks ) );
+			$( '#lang_list' ).on( 'notifyChange', function( event, language ) {
+				// Refresh the flag field
+				selectmenuFlagList.element.val( language.values[3] );
+				selectmenuFlagList._trigger( 'change' );
```

Comment:
Do we need to use jQuery's `Widget._trigger()` function here, or can we access the `change()` callback directly?

Calling it this way might trigger all other event listeners registered on javascript's native `change` event, but is it what we want?

Reply:
I think it's not possible. Callbacks are not method and are code to be triggered on a event.
There is no `change()` method in the jQuery widget factory 
https://api.jqueryui.com/jQuery.widget/
Widget._trigger() is, in my opinion, the right way to do this.